### DISCUSSION TITLE
Panorama width slider

### DISCUSSION THREAD

**Discussion by HydraheadHunter** [AprilSylph/XKit-Rewritten#1034] [Suggestion Box]
I'd love for my tumblr dashboard to be wider, but the panorama feature as written makes it too wide with no control. A width slider option in the panorama's options which controls how wide the panorama mode makes the dash when enabled would not go unappreciated.

**Comment by marcustyphoon** [comment]
Sort of a duplicate of #814, though I do like the idea of a slider!
